### PR TITLE
docs(master-branch): describe branch usage & fix links

### PR DESCRIPTION
**Pull request by Krijger**
- describe branch usage now that develop has been decommissioned and master changed to only accept peer reviewed merge requests
- fix links that pointed to files on develop branch
- some boy scouting on 120 char line length

Interested by the way in how the "reviewers" requesting works. Should be able to be merged after 1 review, so hope it will not wait for @ralphboeije , @wouterd AND @jplattel :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+  referenced using git tags. No direct commits on master are allowed. Instead, all commits are done on feature branches
+  and are peer reviewed before being merged into master.
+- all other branches are considered feature/fix branches.
+
+## Feature branches
+
+- Always create a feature branch with a descriptive name (e.g., `git checkout -b feat-notifications`) when developing  
+  features. Push new commits to your branch only.
+- Keep your branch up to date with the latest commits on the dev branch by occassionally merging in changes
+  `git merge dev`.
```
Comment (wouterd) [inline CONTRIBUTING.md]: This should say "master" branch.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+- When you are finished with your functionality, rebase the branch onto the latest commit of `develop` using the command
```
Comment (wouterd) [inline CONTRIBUTING.md]: Same here, should reference "master"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
+  `git rebase origin/develop` and create a pull request.
+- When an integrator is satisfied with the branch and all feedback is processed, the integrator merges the feature
+  branch into `develop` using `git merge --no-ff feat-notifications`.
```
Comment (wouterd) [inline CONTRIBUTING.md]: Again, should say "master"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,40 +1,45 @@
...
+semantic versioning.  This leads to **more readable messages** that are easy to follow when looking through the
+**project history**.  But also, we use the git commit messages to **generate the change log**.
 
 
 ## Contributing workflow
 
 **Make sure you run `npm install`**  (part-up root folder).
 
-1. Create a **feature-** or **fix-** branch
+1. Create a **feature** (`feat-...` or `ft-`) or **fix** (`fix-...`) branch
```
Comment (wouterd) [inline CONTRIBUTING.md]: We should settle on "ft-" or "feat-" for features.

Comment (Krijger) [inline CONTRIBUTING.md]: I agree, but that is not part of this change. Anyway, "feature-" is just wrong, so this is an improvement until we pick that up.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Krijger) [inline CONTRIBUTING.md]: I used the search functionality of github. Good to know that does not give all the results!

**Comment by Krijger** [comment]
I used the search functionality of github. Good to know that does not give all the results! Updated.

File: CONTRIBUTING.md (hunk 4 above)
Comment (Krijger) [inline CONTRIBUTING.md]: @wouterd this is currently blocking the PR. Pls dismiss this piece of review.

Comment (ralphboeije) [inline CONTRIBUTING.md]: I would suggest feat, as ft is too abbreviated and harder to understand.

**Comment by ralphboeije** [comment]
mmm, reviewers seems to be using the 'AND' operator and not 'OR'...

File: CONTRIBUTING.md (hunk 4 above)
Comment (Krijger) [inline CONTRIBUTING.md]: I like either "ft" or "feature", and believe "feat" is ugly. :) But still - not in scope of this change

**Comment by Krijger** [comment]
@ralphboeije Apparently, on reviewer that blocks actually blocks :) As you would expect. The reviewers does use OR logic though. Because you said OK, I've now been able merge by dismissing @wouterd his last review comment.
